### PR TITLE
Implement multiple capture mode

### PR DESCRIPTION
**Pull request by mlsdpk**
I've implemented the issue #12 (feature multiple capturing mode). Please review guys. @aungmcs @Awthura @aungpaing98

### CODE REVIEW

**Comment by myogh** [comment]
Nice work @mlsdpk, it almost worked.
Upon testing, I found a bug. So if there are multiple opponent pieces available to be captured, the last valid move's location should only be displayed and allowed to press instead of the in-between locations. 
Otherwise, the player could press the middle valid location which I think could go against the rule of the game.

checkerlog

**Comment by mlsdpk** [comment]
> Nice work @mlsdpk, it almost worked.
> Upon testing, I found a bug. So if there are multiple opponent pieces available to be captured, the last valid move's location should only be displayed and allowed to press instead of the in-between locations.
> Otherwise, the player could press the middle valid location which I think could go against the rule of the game.
> 
> checkerlog

Yes, the bug you mentioned is the one I would like to discuss with the team as I found out in some [checkers online game](https://www.247checkers.com/) they allow users to let them decide whether to capture both pieces or only one. Any ideas for this? @aungmcs @Awthura @aungpaing98

**Comment by myogh** [comment]
Ok then. That rule is good for me. If @Awthura amd @aungpaing98  are fine with it, we could close that issue.

**Comment by Awthura** [comment]
Looks like it's working smooth. find_valid_moves function is generalized as well. No objections this time.

File: checkers/Board.py [hunk 1]
```diff
@@ -83,77 +84,34 @@ def find_valid_pieces(self, player_turn):
...
+        curr_row, curr_col = (
+            row,
+            col) if row is not None and col is not None else self.selected_piece
+
+        # set is used here to avoid addition of same piece multiple times
+        self.piece_set = {(curr_row, curr_col)}
+        self.piece_free_grids = []
+
+        # dict - {(after_captured_grid): [(capture_piece), (parent_grid)]}
+        self.capture_pieces = {}
```
Comment (myogh) [inline checkers/Board.py:108]: Shouldn't these instance attributes be declared inside the constructor first? @mlsdpk

Comment (mlsdpk) [reply checkers/Board.py:108]: Yes, sure. I will update it accordingly.

**Pull request merged**